arangod/Agency/Store.cpp

```diff
@@ -365,6 +365,18 @@ std::vector<bool> Store::applyLogEntries(arangodb::velocypack::Builder const& qu
   return applied;
 }
 
+struct slice_equals {
```

Comment:
need to use `3rdParty/velocypack/include/velocypack/Compare.h` and `normalizedEqual`